### PR TITLE
Pivotal -> VMware

### PR DESCRIPTION
**Pull request by chenbh**
Minor details I noticed during my onboarding

Todo:
- [x] figure out how to get GlobalProtect from official sources
- [x] document how to create an Okta account for someone else
- [x] cleanup the `Deployments` section (mainly metrics links)
- [x] follow up with Salesforce, PagerDuty, and interview calendar
- [ ] figure out what to do with the `pivotal/concourse-ops` wiki checklist

### CODE REVIEW

File: sections/checklist.lit [hunk 1]
```diff
@@ -3,9 +3,6 @@
 If you've just joined the team...:
 
 \list{
-  Ask someone to invite you to the \code{concourse-private} channel on Pivotal
```
Comment (xtreme-sameer-vohra) [inline sections/checklist.lit:6]: Since we have `concourse-private` shared with VMware Slack, this can stay up here with just re-naming to VMware

Comment (xtreme-sameer-vohra) [reply sections/checklist.lit:6]: nvm, need to send the HelpNow ticket prior to being access slack

File: sections/checklist.lit [hunk 2]
```diff
@@ -18,11 +15,25 @@ If you've just joined the team...:
   question or just casual discussion - prefer Discord. Lots of interesting
   conversations spark just from being more active in the community.
 }{
-  Have a team member send an email to
-  \link{ask@pivotal.io}{mailto:ask@pivotal.io} like so:
+  <<<<TODO>>>>
+
+  Ask someone to setup an \link{Okta}{https://pivotal.okta.com/app/UserHome} account for you.
```
Comment (xtreme-sameer-vohra) [inline sections/checklist.lit:20]: Spoke with Joden and we can add the request for Okta into the HelpNow ticket

File: sections/checklist.lit [hunk 3]
```diff
@@ -32,19 +43,23 @@ If you've just joined the team...:
...
     - the GitHub organization 'pivotal' (if not present)
-    - the Salesforce and PagerDuty Okta apps
+    - the GitHub team 'pivotal/concourse' (if not present)
 
     Thanks!
     The Concourse Team
   }}
 }{
-  Ask someone to invite you to the following GitHub teams:
+  Ask someone to invite you to the \code{concourse-private} channel on Pivotal
```
Comment (xtreme-sameer-vohra) [inline sections/checklist.lit:59]:
```suggestion
  Ask someone to invite you to the \code{concourse-private} channel on VMware
```

File: sections/checklist.lit [hunk 4]
```diff
@@ -18,11 +15,25 @@ If you've just joined the team...:
...
 }{
-  Have a team member send an email to
-  \link{ask@pivotal.io}{mailto:ask@pivotal.io} like so:
+  <<<<TODO>>>>
+
+  Ask someone to setup an \link{Okta}{https://pivotal.okta.com/app/UserHome} account for you.
+  Okta is needed to be added to the `pivotal` GitHub org as well as creating a
+  G Suite account to access GCP.
+}{
+  If you are working remotely you should make sure you have the GlobalProtect VPN installed.
```
Comment (xtreme-sameer-vohra) [inline sections/checklist.lit:24]: Should be able to get to it at `https://gpu.vmware.com` using the RSA login

Comment (xtreme-sameer-vohra) [inline sections/checklist.lit:24]:
```suggestion
  If you are working remotely you should make sure you have the GlobalProtect VPN installed. Details can be found [here](https://source.vmware.com/portal/pages/IT/global-protect-installation-and-configuration)
```

File: sections/checklist.lit [hunk 5]
```diff
@@ -53,28 +68,27 @@ If you've just joined the team...:
...
-    which will grant you access to any private repos managed by the team (kept
-    under the \code{pivotal} GitHub org).
   }
 
   Remember to check for your invite at the \link{Concourse GitHub
   org}{https://github.com/concourse} and \link{Pivotal GitHub
   org}{https://github.com/pivotal}!
 }{
-  Ask someone to add you to the following Google calendars:
+  Ask someone to add you to the team mailing list \code{pvtl-concourse@vmware.com}
```
Comment (xtreme-sameer-vohra) [inline sections/checklist.lit:77]:
```suggestion
  Ask someone to add you to the team mailing list \code{concourse@vmware.com}
```

We got this simpler one now

File: sections/checklist.lit [hunk 6]
```diff
@@ -32,19 +43,23 @@ If you've just joined the team...:
...
 
     Thanks!
     The Concourse Team
   }}
 }{
-  Ask someone to invite you to the following GitHub teams:
+  Ask someone to invite you to the \code{concourse-private} channel on Pivotal
+  Slack.
+}{
+  Ask someone on the team to invite you to the following GitHub teams:
```
Comment (xtreme-sameer-vohra) [inline sections/checklist.lit:62]: Could we consolidate all the team asks for github teams, calendar and mailing list into one request to concourse@vmware.com ?

Comment (chenbh) [reply sections/checklist.lit:65]: _Outlook would like a Word with you_

Think you tried and it wasn't pretty?

Comment (xtreme-sameer-vohra) [reply sections/checklist.lit:65]: the outlook calendar continues to be extra challenging. however

**Pull request merged**